src/types/guilds/membership_screening_field.ts

```diff
@@ -0,0 +1,17 @@
+import { SnakeCaseProps } from "../util.ts";
+import { MembershipScreeningFieldTypes } from "./membership_screening_field_types.ts";
+
+export interface MembershipScreeningField {
+  /** The type of field (currently "TERMS" is the only type) */
+  fieldType: MembershipScreeningFieldTypes;
+  /** The title of the field */
+  label: string;
+  /** The list of rules */
+  values?: string[];
+  /** Whether the user has to fill out this field */
+  required: boolean;
+}
+
```

Comment:
Missing "doc resource" link here.

Reply:
Does not exist as of now. They have removed the doc. But it still exists, so should we keep the type?

Reply:
Same in this case.